### ISSUE TITLE
Lazy-loading is not working 

### ISSUE THREAD

**Issue by bairaginath** [asafdav/ng-csv#89]
$scope binding does not work on lazy loading if i fetch data from server.

**Comment by bairaginath** [comment]
before binding data ,it downloaded file

**Comment by rajansoft1** [comment]
are you passing promise to it?

**Comment by bairaginath** [comment]
yes Rajan , i already use promise . actually problem is before data binding csv file is downloaded.so in this case i use again click event fired after completed data binding.if u have any idea on please let me know . advance thanks .........

**Comment by rajansoft1** [comment]
can you write a plunkr to show me what you are doing, and can you also show me the data contract of your response?
